README.md

```diff
@@ -606,6 +606,7 @@ These functions provide a very clean way to build out very simple functions and
 | `filter` | <code>((a -> Boolean) &#124; Pred a) -> m a -> m a</code> |
 | `first` | `m (a -> b) -> m (Pair a c -> Pair b c)` |
 | `fold` | `Semigroup s => m s -> s` |
+| `fork` | `Async m => (a -> _) -> (b -> _) -> () -> m a b -> _` |
```

Comment:
Now that I am putting a lot of thought into this. As this is a side-effect, what if `fork` acted like `tap`? So instead of returning the cancel function, what if it just returns the Async back.

```js
function fork(rej, res, m) {
  m.fork(rej, res)
  return m
}
```